### PR TITLE
feat(button): add icon_small size

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Added `icon_small` size, VRT, and test
- [x] Added examples to doc page

### CODE REVIEW

File: packages/paste-website/src/pages/components/button/index.mdx [hunk 1]
```diff
@@ -117,61 +117,51 @@ screen readers and dictation software. Here are some quick tips:
 ### Examples
 
 <LivePreview scope={{Button, Box}} language="jsx">
-  {`<div>
+  {`<Box>
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/button/index.mdx:120]: If you're going to the effort of removing the div, should you convert to use stack on these?

File: packages/paste-core/components/button/src/index.tsx [hunk 2]
```diff
@@ -70,9 +70,12 @@ const handlePropValidation = ({as, href, tabIndex, variant, size, fullWidth, chi
   if (variant === 'reset' && size !== 'reset') {
     throw new Error('[Paste: Button] The "RESET" variant can only be used with the "RESET" size.');
   }
-  if (size === 'icon' && fullWidth) {
+  if ((size === 'icon' || size === 'icon_small') && fullWidth) {
     throw new Error('[Paste: Button] Icon buttons should not be fullWidth.');
   }
+  // if (size === 'icon_small' && fullWidth) {
+  //   throw new Error('[Paste: Button] Icon buttons should not be fullWidth.');
+  // }
```
Comment (SiTaggart) [inline packages/paste-core/components/button/src/index.tsx:78]: assuming you'll remove the comments

File: packages/paste-core/components/button/src/styles.ts [hunk 3]
```diff
@@ -60,6 +60,10 @@ export const SizeStyles: {[key in ButtonSizes]: BoxStyleProps} = {
     padding: 'space30',
     borderRadius: 'borderRadius20',
   },
+  icon_small: {
+    padding: 'space20',
+    borderRadius: 'borderRadius20',
+  },
```
Comment (SiTaggart) [inline packages/paste-core/components/button/src/styles.ts:66]: Pretty cool that's pretty much all you need to do...

File: packages/paste-core/components/button/src/index.tsx (hunk 2 above)
Comment (richbachman) [reply packages/paste-core/components/button/src/index.tsx:78]: Yep. Thanks for the reminder.

File: packages/paste-website/src/pages/components/button/index.mdx [hunk 4]
```diff
@@ -116,62 +116,32 @@ screen readers and dictation software. Here are some quick tips:
...
-  {`<div>
-  <Box display="inline-block" marginRight="space30">
-    <Button variant="primary">
-      ✊ Action
-    </Button>
-  </Box>
-
-<Box display="inline-block" marginRight="space30">
+<LivePreview scope={{Button, Stack}} language="jsx">
+  {`<Stack orientation="horizontal" spacing="space30">
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/button/index.mdx:120]: Nice 👍

**Review by TheSisb — APPROVED** [review]

**Pull request merged**